.github/workflows/modern-tests.yml

```diff
@@ -0,0 +1,82 @@
+name: Modern Tests
```

Comment:
i'm not sure if "modern" would be a good name since in a X moment it can't be modern any amore